Error when an entry is posted from stdin

When I try to post an entry from stdin, an error occured like this.

```
panic: runtime error: invalid memory address or nil pointer dereference
[signal 0xb code=0x1 addr=0x0 pc=0x4149]

goroutine 1 [running]:
main.(*Entry).atom(0xc20803d2f0, 0x2a)
        /Users/naoty/src/github.com/motemen/blogsync/entry.go:103 +0xe9
main.(*Broker).PostEntry(0xc20802b0b0, 0xc20803d2f0, 0x0, 0x0)
        /Users/naoty/src/github.com/motemen/blogsync/broker.go:138 +0x1a9
main.func·003(0xc2080781c0)
        /Users/naoty/src/github.com/motemen/blogsync/main.go:142 +0x567
github.com/codegangsta/cli.Command.Run(0x3e6b30, 0x4, 0x0, 0x0, 0x0, 0x0, 0x0, 0x429590, 0x1a, 0x0, ...)
        /Users/naoty/src/github.com/codegangsta/cli/command.go:118 +0x104b
github.com/codegangsta/cli.(*App).Run(0xc2080a8000, 0xc20800a000, 0x4, 0x4, 0x0, 0x0)
        /Users/naoty/src/github.com/codegangsta/cli/app.go:154 +0xd04
main.main()
        /Users/naoty/src/github.com/motemen/blogsync/main.go:20 +0x149
```

* OS: OS X mavericks
* Go: 1.4.2

--- a/entry.go
+++ b/entry.go
@@ -100,7 +100,9 @@ func (e *Entry) atom() *atom.Entry {
 		Content: atom.Content{
 			Content: e.Content,
 		},
-		Updated: e.Date.Time,
+	}
+	if e.Date != nil {
+		atomEntry.Updated = e.Date.Time
 	}
 
 	if e.IsDraft {

